prep-html-syntax/index.html

```diff
@@ -0,0 +1,13 @@
+<DOCTYPE html>
+  <html>
+    <title></title>
```

Comment:
Be sure to put meta information, such as `<title>` inside of `<head></head>` tags.